src/scripts/sniperino.ts

```diff
@@ -0,0 +1,101 @@
+// Description:
+//  Beat the sniperino
+//  
+//
+// Commands:
+//  sniperino( me)? - start a new game of sniperino 
+//  snipe( me)? - roll a snipe roll in an attempt to beat the sniperino
+//
+// Author:
+//  Steve Shipsey
+
+import { Robot } from "tsbot";
+
+interface ISniper {
+    name: string;
+    gamesPlayed: number;
+    gamesWon: number;
+    currentSnipe?: number;
+
+    getWinRate(): number;
+    getNewSnipe();
+}
+
+class Sniper implements ISniper {
+    public name: string;
+    public gamesPlayed: number;
+    public gamesWon: number;
+    public currentSnipe: number;
+    
+    constructor(n: string) {
+        this.name = n.slice(0, n.indexOf(" "));
+        this.gamesPlayed = 0;
+        this.gamesWon = 0;
+        //this.currentSnipe = this.getNewSnipe();
+    }
+
+    public getWinRate(): number {
```

Comment:
hm? that function does some math. I think it should maybe be renamed "calculateWinRate()"